src/utils/dashboard.js

```diff
@@ -234,7 +235,8 @@ const Dashboard = {
     datetime,
     DirectoryBrowser,
     dialogHelper,
-    itemIdentifier
+    itemIdentifier,
+    setBackdropTransparency
```

Comment:
I'm not a fan of the `Dashboard` global as a whole, but this seems like the best place for now? I would be  curious what others think about this one.